docs/examples/cloudschedulersource/README.md

```diff
@@ -17,7 +17,7 @@ scheduled events from
    [`CloudSchedulerSource`](cloudschedulersource.yaml)
 
    ```shell
-   export APP_ENGINE_LOCATION=us-central1
+   export APP_ENGINE_LOCATION=us-central
    gcloud app create --region=$APP_ENGINE_LOCATION
```

Comment:
Does app engine accept us-central1 as the region? App Engine calls it us-central.

Reply:
No, app engine doesn't accept us-central1 as the region.